### PR TITLE
Set use_late_start_logging=False if stdin is None

### PR DESCRIPTION
**Pull request by xywei**
This should fix the issue described in #15.

### CODE REVIEW

File: pytools/__init__.py [hunk 1]
```diff
@@ -2182,7 +2182,7 @@ def __init__(  # pylint: disable=too-many-arguments
         # feature for interactive use, do not do it unless there is (weak)
         # evidence of interactive use.
         import sys
-        use_late_start_logging = sys.stdin.isatty()
+        use_late_start_logging = bool(sys.stdin and sys.stdin.isatty())
```
Comment (inducer) [inline pytools/__init__.py]: This is super hard to read IMO---or easy to read incorrectly. Just spell it out:
```py
if sys.stdin is None:
    # Can happen, e.g., if pudb is controlling the console.
    use_late_start_logging = False
else sys.stdin.isatty():
    use_late_start_logging = sys.stdin.isatty()
```

Comment (xywei) [reply pytools/__init__.py]: OK, I have replaced the line.

**Comment by xywei** [comment]
The CI failures seem to be caused by networking errors when trying to downloading the script.

**Comment by inducer** [comment]
Whoops. Looks like that ran *just* as I was updating gitlab.

**Comment by inducer** [comment]
Rerunning now.

**Comment by inducer** [comment]
Thanks!

**Pull request merged**